docs/src/formulation.md

```diff
@@ -18,12 +18,14 @@ The complete mathematical formulation, including variable temporal resolutions,
-| $\mathcal{A}^{\text{se}}$ | Storage energy assets with energy method   |          | $\mathcal{A}^{\text{se}} \subseteq \mathcal{A}^{\text{s}}$                                   | This set contains storage assets that use investment energy method (`storage_method_energy = true` in the file [`assets-data.csv`](@ref schemas)). Otherwise fixed energy-to-power ratio method is used.                             |
-| $\mathcal{A}^{\text{sb}}$ | Storage energy assets with binary method   |          | $\mathcal{A}^{\text{sb}} \subseteq \mathcal{A}^{\text{s}} \setminus \mathcal{A}^{\text{ss}}$ | This set contains storage assets that use an extra binary variable to avoid charging and discharging simultaneously (`use_binary_storage_method` set to `binary` or `relaxed_binary` in the file [`assets-data.csv`](@ref schemas)). |
+| Name                         | Description                                       | Elements | Superset                                                                                     | Notes                                                                                                                                                                                                                                |
+| ---------------------------- | ------------------------------------------------- | -------- | -------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| $\mathcal{A}^{\text{i}}$     | Energy assets with investment method              |          | $\mathcal{A}^{\text{i}}  \subseteq \mathcal{A}$                                              |                                                                                                                                                                                                                                      |
+| $\mathcal{A}^{\text{ss}}$    | Storage energy assets with seasonal method        |          | $\mathcal{A}^{\text{ss}} \subseteq \mathcal{A}^{\text{s}}$                                   |                                                                                                                                                                                                                                      |
+| $\mathcal{A}^{\text{se}}$    | Storage energy assets with energy method          |          | $\mathcal{A}^{\text{se}} \subseteq \mathcal{A}^{\text{s}}$                                   | This set contains storage assets that use investment energy method (`storage_method_energy = true` in the file [`assets-data.csv`](@ref schemas)). Otherwise fixed energy-to-power ratio method is used.                             |
```

Comment:
These notes about the code are crossing the line between Explanation (read in the bath) and Reference/How-To (look something up while implementing).  Can we do something like put a link here and explain this in a Reference/How-To section?

Reply:
But if you want to update and publish you can leave it for now and open an issue to fix it. :)

Reply:
I have changed it to reference the how-to section. I think we should centralize all these parameters setup in that section. That might be one the action points in the full review of the documentation.